### DISCUSSION TITLE
Best way to manage many Tesla?

### DISCUSSION THREAD

**Discussion by vbarrier** [teslamate-org/teslamate#3207] [General]
Hi! 

We use Teslamate to help us manage 20 Tesla at the moment. It seems that we reach a max as we receive a lot of errors from the Tesla API (429 & 408 mainly). Is there a maximum that a Tesla can manage? 

Any workaround? Could we use several Tesla account on the same Teslamate server?

Cheers!

**Comment by cwanja** [comment]
Are these all under the same Tesla account? 

It seems like a limitation for Tesla and less so TeslaMate. After all, this is an unsupported, unofficial API endpoint. And so Tesla accommodating a 'support' request is a dead end.

**Reply by vbarrier** [reply]
Yes they are on the same account :)

**Reply by cwanja** [reply]
A thought would be to create a single account for each vehicle and then add a 'main' account as a secondary driver. Not sure how that would work for your rental operations. But not sure how else to get around Tesla's limitations. 

Are the errors preventing data from being collected? Or are they just errors?

**Reply by vbarrier** [reply]
It doesn't prevent to collect data but a lot of logs and sometimes a delay is visible.
